src/components/question.cjsx

```diff
@@ -33,7 +33,10 @@ module.exports = React.createClass
     hasCorrectAnswer = !! @props.correct_answer_id
 
     if @props.feedback_html
-      feedback = <ArbitraryHtml className='question-feedback has-html' html={@props.feedback_html} />
+      feedback = <ArbitraryHtml
+        className='question-feedback has-html'
+        html="<div class='question-feedback-content'>#{@props.feedback_html}</div>"
```

Comment:
Why is this appending HTML in the string instead of something like:

```coffee
feedback = <div className='question-feedback'>
  <ArbitraryHtml
    className='question-feedback-content has-html'
    html={@props.feedback_html}
    block={true}/>
</div>
```